apps/trustlab/src/lib/data/common/index.js

```diff
@@ -142,7 +179,13 @@ export async function getPageProps(api, context) {
     }
     return null;
   }
-  return {};
+  const siteSettings = await api.findGlobal("site-settings");
+  const navbar = getNavBar(siteSettings);
+  const footer = getFooter(siteSettings);
```

Comment:
Something to think about, can't we use after-read hook or something on payload to do all this massaging and return data in usable format.

Reply:
That's why I said "something to think about"... I don't expect you to do it in this PR (or even in this sprint).